converter/diag/report.go

```diff
@@ -6,14 +6,47 @@ import (
 
 const Text = ".txt"
 
-// generateTextReport generates a text report for the diagnostics.
-func generateTextReport(writer io.Writer, ds Diagnostics) error {
-	content := ds.Error()
+const criticalErrorFooter = `
+
+A configuration file was not generated due to critical issues. Review the diagnostics above.`
+
+const errorFooter = `
+
+A configuration file was not generated due to errors. Review the diagnostics above.
```

Comment:
```suggestion
Grafana Agent couldn't generate a configuration file. Refer to the error message for more information.
```

If we switch things around, we make the information active vs passive, and are specific about who couldn't do what.

What is output? is it actually diagnostics output? or just an error message? We should try to avoid directions (above/below) and be explicit about where the user needs to look.